test/integration/scheduler/scheduler_test.go

```diff
@@ -461,7 +461,6 @@ var _ = ginkgo.Describe("Scheduler", func() {
 			gomega.Expect(k8sClient.Create(ctx, onDemandFlavor)).Should(gomega.Succeed())
 
 			cq = testing.MakeClusterQueue("cluster-queue-with-selector").
-				QueueingStrategy(kueue.StrictFIFO).
```

Comment:
This seems problematic (but beyond the scope of this PR): if a namespace doesn't match the CQ, it shouldn't block other workloads, even if StrictFIFO.

Reply:
Right, perhaps those workloads shouldn't be added to the CQ cache at all, we should add them only when the namespace matches. This means that those workloads wouldn't be counted in the CQ metrics, which I think is reasonable.

Reply:
But I think even we don't put it into the CQ cache, we still need to tell the message(by event) to the User why this workload can't be scheduled successfully(not match).

Reply:
we can do that in the workload controller, just like for cases where the CQ is not existent.